<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Development</class>
 <widget class="QDialog" name="Development">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plant Development</string>
  </property>
  <widget class="QLabel" name="lbl_dG">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>384</width>
     <height>216</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>dG</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Gn1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>384</width>
     <height>216</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Gn1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Gn2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>384</width>
     <height>216</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Gn2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>260</y>
     <width>301</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>281</width>
      <height>126</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="ButtonBox">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="comboBox_start_Date"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_end_Date"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btn_back">
         <property name="text">
          <string>← &amp;Back</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_next">
         <property name="text">
          <string>&amp;Next →</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="lbl_info_1">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhMultiLine</set>
       </property>
       <property name="text">
        <string>Img1 to Img2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_info_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhMultiLine</set>
       </property>
       <property name="text">
        <string>xxx Images</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_exit">
       <property name="text">
        <string>&amp;Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lbl_text">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>plant development range</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lbl_img_start">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>384</width>
     <height>216</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_img_end">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>250</y>
     <width>384</width>
     <height>216</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>End</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_img_end_dg">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>384</width>
     <height>216</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>End + dG</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkBox_inc_dec">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>540</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>zunahme / abnhame</string>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>lbl_Gn1</zorder>
  <zorder>lbl_Gn2</zorder>
  <zorder>lbl_dG</zorder>
  <zorder>lbl_img_start</zorder>
  <zorder>lbl_img_end</zorder>
  <zorder>lbl_img_end_dg</zorder>
  <zorder>chkBox_inc_dec</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
